### PR TITLE
Send message tester implementation.

### PR DESCRIPTION
**Pull request by Eitu33**

### CODE REVIEW

File: src/interface_impl.rs [hunk 1]
```diff
@@ -146,4 +146,19 @@ impl Interface for InterfaceImpl {
...
+        _target_address: &str,
+        _target_handler: &str,
+        _validity_start: (u64, u8),
+        _validity_end: (u64, u8),
+        _max_gas: u64,
+        _gas_price: u64,
+        _coins: u64,
+        data: &[u8],
+    ) -> Result<()> {
+       println!("Send message data: {:?}", data);
```
Comment (qdrn) [inline src/interface_impl.rs:161]: I think it should rather be `Sent` or `Sending`.

**Comment by Eitu33** [comment]
bors merge

**Comment by Eitu33** [comment]
bors merge

**Comment by qdrn** [comment]
There is no bors on this repo. Simply merge :)

**Pull request merged**